src/cmd/linuxkit/run_gcp.go

```diff
@@ -46,6 +46,7 @@ func runGcp(args []string) {
 
 	skipCleanup := flags.Bool("skip-cleanup", false, "Don't remove images or VMs")
 	nestedVirt := flags.Bool("nested-virt", false, "Enabled nested virtualization")
+	vTPM := flags.Bool("vTPM", false, "Enabled vTPM device")
```

Comment:
I am concerned that a flag like `--vTPM` is going to lead to case errors. It may not be as pretty, but making it all lower-case might be more reliable; it certainly would be more consistent.

The var itself, of course, can remain `vTPM`.

> "Enabled vTPM device"

I think this should be "Enable vTPM device". Granted, the one before also says "Enabled", but it shouldn't. This is imperative ("go and enable this"), not descriptive ("the device already was enabled").

Reply:
Make sense, modified